package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "frontcamp",
+  "version": "1.0.0",
+  "description": "FrontCamp Mentoring Program BY Q4 2018",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/PolinaBobrovnik/FrontCamp.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/PolinaBobrovnik/FrontCamp/issues"
+  },
+  "homepage": "https://github.com/PolinaBobrovnik/FrontCamp#readme",
+  "babel": {
+    "presets": [
+      "latest"
+    ]
+  },
+  "devDependencies": {
+    "babel-preset-latest": "^6.16.0",
+    "del": "^2.2.2",
+    "gulp": "^3.9.1",
+    "gulp-babel": "^6.1.2",
+    "gulp-concat": "^2.6.1"
+  },
+  "dependencies": {
+    "babel-preset-es2015": "^6.6.0",
```

Comment:
babel, gulp are dev depth, everything except `bootstrap` and `moment`

additionally I recommend avoid usage `^`